zio-http/src/main/scala/zio/http/URL.scala

```diff
@@ -73,11 +75,11 @@ final case class URL(
     copy(kind = location)
   }
 
-  def setQueryParams(queryParams: Map[String, List[String]]): URL =
+  def setQueryParams(queryParams: QueryParams): URL =
```

Comment:
I think we should keep the older API also, and add this as an overload.
We can even add  `(String, List[String]) *` as an overload.

Reply:
Done